_automating-configurations/workflow-steps.md

```diff
@@ -42,6 +42,7 @@ The following table lists the workflow step types. The `user_inputs` fields for
 |`create_index`|[Create Index]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/)     | Creates a new OpenSearch index. The inputs include `index_name`, which should be the name of the index to be created, and `configurations`, which contains the payload body of a regular REST request for creating an index.
 |`create_ingest_pipeline`|[Create Ingest Pipeline]({{site.url}}{{site.baseurl}}/ingest-pipelines/create-ingest/) | Creates or updates an ingest pipeline. The inputs include `pipeline_id`, which should be the ID of the pipeline, and `configurations`, which contains the payload body of a regular REST request for creating an ingest pipeline.
 |`create_search_pipeline`|[Create Search Pipeline]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/creating-search-pipeline/) | Creates or updates a search pipeline. The inputs include `pipeline_id`, which should be the ID of the pipeline, and `configurations`, which contains the payload body of a regular REST request for creating a search pipeline.
+|`reindex`|[Reindex]({{site.url}}{{site.baseurl}}/api-reference/document-apis/reindex/)  | The reindex document API operation lets you copy all or a subset of your data from a source index into a destination index. The input includes `source_indices`, `destination_index` and few additional optional parameters like `refresh`, `requests_per_second`, `require_alias`, `slices` and `max_docs`. Note: Using this workflow step can make the cluster unstable.
```

Comment:
> we don't trigger a flush for every upload if thats what you were asking

You might want to add things like ensure your cluster and model is properly scaled to handled the re-indexing workload. Use the parameters provided to control the request rate.(requests_per_second).


The problem I am seeing is we just wrote in doc it will make cluster unstable but didn't write why and how to ensure the cluster is not getting unstable. 

This needs rework.